fastpair/test/test_fastpair.py

```diff
@@ -168,24 +164,24 @@ def test_all_closest_pairs(self, PointSet):
         # Ordering may be different, but both should be in there
         # assert dc[1][0] in cp[1] and dc[1][1] in cp[1]
 
-    def test_find_neighbor_and_sdist(self, PointSet):
-        ps = PointSet
+    def test_find_neighbor_and_sdist(self, point_set):
+        ps = point_set
         fp = FastPair().build(ps)
         rando = rand_tuple(len(ps[0]))
         neigh = fp._find_neighbor(rando)  # Abusing find_neighbor!
         dist = fp.dist(rando, neigh["neigh"])
         assert abs(dist - neigh["dist"]) < 1e-8
         assert len(fp) == len(ps)  # Make sure we didn't add a point...
-        l = [(fp.dist(a, b), b) for a, b in zip(cycle([rando]), ps)]
-        res = min(l, key=itemgetter(0))
+        l_ = [(fp.dist(a, b), b) for a, b in zip(cycle([rando]), ps)]
```

Comment:
why the underscore here?

Reply:
This is due to:

```
fastpair/test/test_fastpair.py:179:9: E741 Ambiguous variable name: `l`
```

If you prefer the `l` variable name, we can keep that and I'll throw a `# noqa: E741` in there to stop `ruff` from hollering.

Reply:
naw it doesn't matter